modules/lvms-provisioning-storage-using-logical-volume-manager-operator.adoc

```diff
@@ -8,6 +8,12 @@
 
 After you have created the LVM volume groups using the `LVMCluster` custom resource (CR), you can provision the storage by creating persistent volume claims (PVCs).
 
+The minimum storage size that you can request for each file system type is as follows:
+
+* `Filesystem`: 300 MiB
```

Comment:
```suggestion
* `xfs`: 300 MiB
```

Reply:
looking at this again, i suggest moving block above, and lower-casing it.

Reply:
Updated as per the suggestions.